pulp_python/app/models.py

```diff
@@ -23,51 +23,42 @@ class Classifier(models.Model):
     """
 
     name = models.TextField()
-    pythonpackagecontent = models.ForeignKey("PythonPackageContent", related_name="classifiers",
+    python_package_content = models.ForeignKey("PythonPackageContent", related_name="classifiers",
                                              related_query_name="classifier")
 
 
 class PythonPackageContent(Content):
     """
     A Content Type representing Python's Distribution Package as
-    defined in pep-0426
+    defined in pep-0426 and pep-0345
+    https://www.python.org/dev/peps/pep-0491/
+    https://www.python.org/dev/peps/pep-0345/
     """
 
     TYPE = 'python'
     filename = models.TextField(unique=True, db_index=True, blank=False)
-    path = models.TextField(blank=False)
     packagetype = models.TextField(blank=False)
     name = models.TextField(blank=False)
     version = models.TextField(blank=False)
-
-    # Note that this field is not contained in PyPI nor Warehouse
-    # ignore this on publish to PyPI or Warehouse
-    # This is required per PEP426
     metadata_version = models.TextField(blank=False)
-
     summary = models.TextField()
     description = models.TextField()
-    keywords = models.TextField()
+    keywords = ArrayField(models.TextField())
     home_page = models.URLField()
     download_url = models.URLField()
     author = models.TextField()
-    author_email = models.EmailField()
+    author_email = models.TextField()
```

Comment:
It turns out that a valid email only needs to conform to [RFC822](http://www.faqs.org/rfcs/rfc822.html)
so `Author-email: "C. Schultz" <cschultz@example.com>` is valid for python metadata, but django fails the email validation for it.